doc/guides/west/manifest.rst

```diff
@@ -1179,6 +1189,430 @@ processed in this order:
+ensure this by running ``west config manifest.groups -- -groupA``; the extra
+``--`` is required so the argument parser does not treat ``-groupA`` as a
+command line option ``-g`` with value ``roupA``).
+
+Project ``foo`` is inactive because ``groupA`` has been blocked by the
+``manifest.groups`` configuration option. Project ``bar`` is active because
+``groupB`` is allowed.
+
+Example 5: Overriding a blocked group via configuration
+-------------------------------------------------------
+
+The entire manifest file is:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: foo
+       - name: bar
+         groups:
+           - groupA
+       - name: baz
+         groups:
+           - groupA
+           - groupB
+
+     groups: [-groupA]
+
+     defaults:
+       remote: example-remote
+     remotes:
+       - name: example-remote
+         url-base: https://git.example.com
+
+The ``manifest.groups`` configuration option is set to ``groupA`` (you can
+ensure this by running ``west config manifest.groups groupA``).
+
+In this case, ``groupA`` is allowed: the ``manifest.groups`` configuration
+option has higher precedence than the ``manifest: groups: [-groupA]`` content
+in the manifest file.
+
+Therefore, projects ``foo`` and ``bar`` are both active.
+
+Example 6: Overriding multiple blocked groups via configuration
+---------------------------------------------------------------
+
+The entire manifest file is:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: foo
+       - name: bar
+         groups:
+           - groupA
+       - name: baz
+         groups:
+           - groupA
+           - groupB
+
+     groups: [-groupA,-groupB]
+
+     defaults:
+       remote: example-remote
+     remotes:
+       - name: example-remote
+         url-base: https://git.example.com
+
+The ``manifest.groups`` configuration option is set to ``groupA,groupB``
+(you can ensure this by running ``west config manifest.groups
+"groupA,groupB"``).
+
+In this case, both ``groupA`` and ``groupB`` are allowed, because the
+configuration value overrides the manifest file for both groups.
+
+Therefore, projects ``foo`` and ``bar`` are both active.
+
+Example 7: Blocking multiple groups via configuration
+-----------------------------------------------------
+
+The entire manifest file is:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: foo
+       - name: bar
+         groups:
+           - groupA
+       - name: baz
+         groups:
+           - groupA
+           - groupB
+
+     defaults:
+       remote: example-remote
+     remotes:
+       - name: example-remote
+         url-base: https://git.example.com
+
+The ``manifest.groups`` configuration option is set to ``-groupA,-groupB``
+(you can ensure this by running ``west config manifest.groups --
+"-groupA,-groupB"``).
+
+In this case, both ``groupA`` and ``groupB`` are blocked.
+
+Therefore, projects ``foo`` and ``bar`` are both inactive.
+
```

Comment:
I don't want to be a pain, but I think we should also include an example using the command-line option to `west update` here.